packages/react-component-library/src/components/RangeSlider/RangeSlider.stories.tsx

```diff
@@ -19,6 +19,23 @@ stories.add('Default', () => (
   />
 ))
 
+examples.add('Multiple handles', () => (
```

Comment:
It feels strange this change is here because `thresholds` would not have any impact until it is implemented in a subsequent commit.

Reply:
Ah yeah you are right - this started off as a different commit with a temporary 'MSP' story.

I'll move this commit.